src/apps/viewer/app.ts

```diff
@@ -48,6 +48,7 @@ import '../../mol-util/polyfill';
 import { ObjectKeys } from '../../mol-util/type-helpers';
 import { SaccharideCompIdMapType } from '../../mol-model/structure/structure/carbohydrates/constants';
 import { Backgrounds } from '../../extensions/backgrounds';
+import { StructConnExtensionFunctions } from '../../extensions/struct-conn';
```

Comment:
Can you please move this to the `extensions/wwpdb/struct-conn` folder and rename it to `wwPDBStructConnExtensionFunctions`.

Reply:
OK.